### PR TITLE
Release 2.0.0-beta.2

### PR DESCRIPTION
**Pull request by erinha**
This release contains fixes for SDK API reference documentation.

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : 6345c545559d7f786c4b6a3cd9663987957665d5</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: d4fca89a93d4ee7dc26ada0b2613893953712853</p>

File: azure-pipelines-with-e2etests.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ pool:
 
 steps:
   - checkout: self
-  - checkout: git://$(AppHostingSdkGitPath)@$(AppHostingSdkGitRef)
+  - checkout: git://$(AppHostingSdkGitPath)@f6564541195b32164e1bd21a79f5368dc553600f
```
[the remark this answers is not in the document]
Comment (erinha) [reply azure-pipelines-with-e2etests.yml]: Removed this change

**Comment by erinha** [comment]
<p>Analyzed commit id : 2a662cc0e2f2fcfb61cf9865896d162e019c04de</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>57.34 KB</td><td>57.34 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 95bc08923585f1a9d05f1a25d6190eef635d29c0</p>

**Review by TrevorJoelHarris — APPROVED** [review]
:shipit:

**Pull request merged**